src/h2olog/generated_raw_tracer.cc

```diff
@@ -1774,6 +1774,7 @@ std::string h2o_raw_tracer::bpf_text() {
 
 #include <linux/sched.h>
 #include <linux/limits.h>
+#include "include/h2o/ebpf.h"
```

Comment:
We've avoided including h2o headers in BPF programs because it's a runtime dependency. However, IIRC, this is because h2olog was maintained in the separate repository so that h2olog did not know where h2o was installed. 
Now h2olog can use  `H2O_ROOT`, we should add it to BCC's `cflags` in order to include h2o headers in BPF programs.

Reply:
Nice catch. Addressed in 11c656e.

FWIW, my understanding for not including the header files is because most of our header files cannot be used in eBPF programs. But ebpf.h is specifically written to accommodate such use, hence no reason no prohibit. Anyways, we have consensus that it can be included.